Jenkinsfile

```diff
@@ -139,55 +131,31 @@ pipeline {
         }
         stage("Retag the image if the test passed and delete an old tag") {
             steps{
-                withCredentials([usernameColonPassword(credentialsId: 'quay-aerogear-bot', variable: 'QUAY_CREDS')]) {
-                    retry(3) {
-                        sh """
-                            skopeo copy \
-                              --src-creds ${env.QUAY_CREDS} \
-                              --dest-creds ${env.QUAY_CREDS} \
-                              docker://${env.OPERATOR_CONTAINER_IMAGE_CANDIDATE_NAME} \
-                              docker://${env.OPERATOR_CONTAINER_IMAGE_NAME}
-                        """
-                    }
-                    retry(3) {
-                        sh """
-                            skopeo delete \
-                              --creds ${env.QUAY_CREDS} \
-                              docker://${env.OPERATOR_CONTAINER_IMAGE_CANDIDATE_NAME} \
-                            || sleep 10
-                        """
-                    }
-                }
+                // qe-pipeline-library step
+                tagRemoteContainerImage(
+                    credentialsId: "${env.CREDENTIALS_ID}",
+                    sourceImage: "${env.OPERATOR_CONTAINER_IMAGE_CANDIDATE_NAME}",
+                    targetImage: "${env.OPERATOR_CONTAINER_IMAGE_NAME}",
+                    deleteOriginalImage: true
+                )
             }
         }
         stage("Create a 'latest' tag from 'master'") {
             when {
                 branch 'master'
             }
             steps{
-                withCredentials([usernameColonPassword(credentialsId: 'quay-aerogear-bot', variable: 'QUAY_CREDS')]) {
-                    retry(3) {
-                        sh """
-                            skopeo copy \
-                              --src-creds ${env.QUAY_CREDS} \
-                              --dest-creds ${env.QUAY_CREDS} \
-                              docker://${env.OPERATOR_CONTAINER_IMAGE_NAME} \
-                              docker://${env.OPERATOR_CONTAINER_IMAGE_NAME_LATEST}
-                        """
-                    }
-                }
+                // qe-pipeline-library step
+                tagRemoteContainerImage(
+                    credentialsId: "${env.CREDENTIALS_ID}",
+                    sourceImage: "${env.OPERATOR_CONTAINER_IMAGE_NAME}",
+                    targetImage: "${env.OPERATOR_CONTAINER_IMAGE_NAME_latest}",
+                    deleteOriginalImage: false
+                )
             }
         }
     }
     post {
-        always{
-            script {
-                sh """
-                oc delete project ${env.OPENSHIFT_PROJECT_NAME}
-                rm -rf ${env.CLONED_REPOSITORY_PATH}
-                """
-            }
-        }
```

Comment:
this part was replaced by [this step](https://github.com/aerogear/unifiedpush-operator/pull/30/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R64)